docs/02-guides/display-prices-with-or-without-taxes.mdx

```diff
@@ -0,0 +1,79 @@
+| :-------------------------------: | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
+| `priceInclTax` and `priceExclTax` |    <iframe src="https://fc-3031-3-price-variant-componen.v3-gezy.eu1.front-commerce.dev/styleguide/iframe.html?globals=&args=&id=components-molecules-pricevariant--labeled-incl-tax-excl-tax" style={{ width: "100%", height: "50px", border: "none", }} />    |
+|          `priceInclTax`           |        <iframe src="https://fc-3031-3-price-variant-componen.v3-gezy.eu1.front-commerce.dev/styleguide/iframe.html?globals=&args=&id=components-molecules-pricevariant--labeled-incl-tax" style={{ width: "100%", height: "50px", border: "none", }} />         |
+|          `priceExclTax`           | <iframe src="https://fc-3031-3-price-variant-componen.v3-gezy.eu1.front-commerce.dev/styleguide/iframe.html?globals=&args=&id=components-molecules-pricevariant--labeled-excl-tax&viewMode=story" style={{ width: "100%", height: "50px", border: "none", }} /> |
+
+### Default
+
+|            Price Type             |                                                                                                                                 Example                                                                                                                                  |
+| :-------------------------------: | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
+| `priceInclTax` and `priceExclTax` | <iframe src="https://fc-3031-3-price-variant-componen.v3-gezy.eu1.front-commerce.dev/styleguide/iframe.html?globals=&args=&id=components-molecules-pricevariant--default-incl-tax-excl-tax&viewMode=story" style={{ width: "100%", height: "50px", border: "none", }} /> |
+|          `priceInclTax`           |             <iframe src="https://fc-3031-3-price-variant-componen.v3-gezy.eu1.front-commerce.dev/styleguide/iframe.html?globals=&args=&id=components-molecules-pricevariant--default-incl-tax" style={{ width: "100%", height: "50px", border: "none", }} />             |
+|          `priceExclTax`           |             <iframe src="https://fc-3031-3-price-variant-componen.v3-gezy.eu1.front-commerce.dev/styleguide/iframe.html?globals=&args=&id=components-molecules-pricevariant--default-excl-tax" style={{ width: "100%", height: "50px", border: "none", }} />             |
```

Comment:
The urls should be updated once the feature is merged

Reply:
I think this should also be refactored to use jsx table tags instead of mdx for readability